src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandlerTest.java

```diff
@@ -36,14 +36,14 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "indentation" + File.separator + filename);
+                + "indentation" + File.separator + "newhandler" + File.separator + filename);
     }
 
     @Test
     public void testInvalidLabel() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(IndentationCheck.class);
```

Comment:
Right now it doesn't makes sense, we should merge test to IndentationCheckTest.
It might be created to avoid conflicts on merge a long time ago.